### PR TITLE
Feature/wz1264 personrole

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/components/EventsTimeline/components/EventData.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ const displayValue = (value: unknown, mapValue: Field["mapValue"], isCurrency =
 
   const mappedValue = mapValue(value)
 
+  if(mappedValue === undefined) return null
```
Comment (gilleswittenberg) [inline src/components/EventsTimeline/components/EventData.tsx:29]: Can I ask to add a space after `if`?

File: src/components/Helpers/dictionaries.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const personRoleMap = {
+  "PERSON_ROLE_OWNER": "Eigenaar",
+  "PERSON_ROLE_RESIDENT": "Bewoner",
+  "PERSON_ROLE_MIDDLEMAN": "Tussenpersoon"
+}
```
Comment (gilleswittenberg) [inline src/components/Helpers/dictionaries.ts:5]: Would have added the `PersonRole` type here. Like: `export type PersonRole = keyof typeof personRoleMap`. And remove the export everywhere else.

File: src/components/EventsTimeline/components/EventData.tsx (hunk 1 above)
Comment (remcohoff) [reply src/components/EventsTimeline/components/EventData.tsx:29]: Yes you can, fixed it

File: src/components/EventsTimeline/helpers/DisplayPersonWithRole.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  person_role: PersonRole
+}
+
+export type PersonRole = "PERSON_ROLE_OWNER" | "PERSON_ROLE_RESIDENT" | "PERSON_ROLE_MIDDLEMAN"
+
+const personRole = (role: keyof typeof personRoleMap) => personRoleMap[role]
+
+const DisplayPersonWithRole = (person: Person) => 
+  // TODO use PersonNameDisplay (with person_full_name) and PersonRoleDisplay
+  `${ person.person_full_name }, ${ typeof person.person_role === "string" ? personRole( person.person_role as PersonRole ).toLowerCase() : "-" }`
```
Comment (gilleswittenberg) [inline src/components/EventsTimeline/helpers/DisplayPersonWithRole.tsx:14]: `personRole( person.person_role as PersonRole ).toLowerCase()` can be `personRoleMap[person.person_role].toLowerCase()`

**Review by gilleswittenberg — APPROVED** [review]

**Pull request merged**